app/models/r_package.rb

```diff
@@ -123,8 +123,11 @@ def self.explain_opencpu_configuration
   def self.logged_call(function, parameters)
     Rails.logger.debug "Calling OpenCPU: #{function}(#{parameters})" # Only log non-cached calls
 
-    result = Client.instance.execute 'screensmart', function,
-                                     user: :system, data: parameters, convert_na_to_nil: true
+    result = Appsignal.instrument "screensmart-r.#{function}", function, parameters.to_s do
```

Comment:
`logged_call` wordt alleen aangeroepen als een call niet cached is, dus het gaat maar om een paar verzoekjes.

Kun je uitleggen waarom het je niet gepast lijkt om dit naar Appsignal te loggen?